etna/core/mixins.py

```diff
@@ -80,6 +93,22 @@ def to_dict(self):
         params["_target_"] = BaseMixin._get_target_from_class(self)
         return params
 
+    @staticmethod
+    def _unsafe_to_dict(value: Any) -> Dict[str, Any]:
+        """Collect all information about etna object in dict."""
```

Comment:
It isn't only about etna object, we use it for callbacks from pytorch_lightning.